src/apps/testapps/testPolygon.c

```diff
@@ -45,6 +46,14 @@ GeoCoord transMeridianHoleVerts[] = {{0.005, -M_PI + 0.005},
                                      {-0.005, -M_PI + 0.005}};
 Geofence transMeridianHoleGeofence;
 
+void destroyLinkedGeoLoop(LinkedGeoLoop* loop) {
```

Comment:
I'm a little confused why this is needed in the tests but not in the library code.

Reply:
In the library code, `destroyLinkedGeoPolygon` walks the whole structure and destroys all the loops as well. The rest of the library mostly deals with the polygons, not with the loops, but the new functions take the loops as input, so we're creating them directly in tests - we don't do this in lib code.

Reply:
Can you make these functions static if you don't use them in other C files?